README.md

```diff
@@ -0,0 +1,48 @@
+# Guitar API
+
+## Table of Contents
+- [Description](#description) :page_with_curl:
+- [Requirements](#requirements) :hammer_and_wrench:
+- [Getting Started](#getting-started)
+   - [Installation](#installation)
+- [Documentation](#official-documentation)
+- [Feature Roadmap](#feature-roadmap)
+
+## Description 
+The Guitar API is a RESTful API that provides information about various guitar makes and modes, along with information about famous mucisians who have used those models at some point in their careers. This API allows developers to access ruitar-related information to build applications, websites, or any other projects that require guitar-related data.
+
+## Requirements
+- Go Programming Language
+- PostgreSQL
+   - For storing and retrieving API database information
+- pgAdmin
+- API Testing Tool
+   - [Postman][https://www.postman.com/] or something similar
```

Comment:
```suggestion
   - [Postman](https://www.postman.com/) or something similar
```



<img width="540" alt="CleanShot 2023-06-14 at 18 24 00@2x" src="https://github.com/Bobs-code/Guitar-API/assets/1369770/fa090821-1344-4509-aa7b-a3fbd8e575c8">

I didn’t catch this in the code review, but it stuck out when looking at [the preview](https://github.com/Bobs-code/Guitar-API/tree/readme)